Components/Images/images.css

```diff
@@ -7,7 +7,7 @@
 .img-responsive {
```

Comment:
It's a suggestion to not change styles of other components while working on the branch of another one.

Reply:
Thanks Bharati for mentioning this. I will keep this in mind.